### ISSUE TITLE
Other "New blank Tab" actions also override "New Tab" button behaviour

### ISSUE THREAD

**Issue by mutmedia** [piroor/treestyletab#2412]
## Short description
Changing the behaviour of "Other new tab" actions to make new tabs created with say, ctrl+T, be added as child of the current one also overrides the behaviour of the "New Tab" button. 
## Steps to reproduce
1. Download TST version 3.2.3
2. Change the "Other new tab"  behaviour to be creating tabs as child of the current one
![image](https://user-images.githubusercontent.com/4412564/68260880-2e463680-fff3-11e9-9743-9e4b16ff31db.png)
3. Attempt to open a new tab using the "New Tab" button

## Expected result
"New Tab" button behaviour should remain unchanged, so the new tab should be an independent one

## Actual result
"New Tab" button behaviour is overwritten by the "Other new tab" action, creating the new tab as a child of the current one

**Comment by piroor** [comment]
Sorry for confusion. It is intentional by design. Actually the option `Open "New Blank Tab" actions (toolbar buttons, keyboard shortcuts, etc.)` is applied to new tabs from the "New Tab" button in the sidebar. So it should be named like as `All "Open New Blank Tab" actions including the button itself`...

Inspired from this issue I've switched the order of two sections and renamed those captions. I hope it reduces such misunderstandings.

![image](https://user-images.githubusercontent.com/70062/68269308-ac124c80-009c-11ea-8016-2164a88a6694.png)
![image](https://user-images.githubusercontent.com/70062/68269332-d06e2900-009c-11ea-845a-94d4b0f6922b.png)

In other words currently you cannot specify different result for a new blank tab from the "New Tab" button and a new tab from Ctrl-T.

**Comment by mutmedia** [comment]
Is it too complicated to make these two behaviours different? Also, is there a way to create new keyboard shortcuts?

**Comment by piroor** [comment]
TST provides ability to assign separate keyboard shortcuts for each action about new tabs.
![image](https://user-images.githubusercontent.com/70062/68351789-da993180-0147-11ea-96cd-66111ecfdcbc.png)
But please remind that currently Firefox doesn't allow assign any shortcut conflicting to Firefox's one. (related bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1325692 )

**Comment by mutmedia** [comment]
Awesome! I guess this issue is closed then?

**Comment by piroor** [comment]
Yes, please close this if you don't have any more problem about it. (I recommend reporters to close issues by themselves, because issues closed by the project owner cannot be reopened by the reporter.)
